src/main/webapp/resources/css/app.css

```diff
@@ -91,42 +91,6 @@ body > .ant-layout {
   background-color: #ffffff;
 }
 
-/****************************************************************************************
-* BOOTSTRAP OVERRIDES
-*****************************************************************************************/
-.nav,
-.pagination,
-.carousel,
-.panel-title a {
-  cursor: pointer;
-}
-
-.nav-tabs {
-  margin-bottom: 1.3rem;
-  border-bottom: 1px solid #e6e8eb !important;
-
-  li {
-    a {
-      border: none !important;
-    }
-
-    & .active a {
-      font-weight: 600;
-      background-color: transparent !important;
-      border-bottom: 1px solid #1c2833 !important;
-    }
-
-    & :not(.active) a {
-      color: #717f8c !important;
-      background-color: transparent !important;
-    }
-  }
-}
-
-label.ant-checkbox-wrapper {
```

Comment:
Was this mistakenly removed, this is set for `.ant-checkbox-wrapper`

Reply:
Was there to override bootstraps checkbox css